lisp/ess-inf.el

```diff
@@ -1273,7 +1275,7 @@ wrapping the code into:
   (let ((out-buffer (or out-buffer (get-buffer-create " *ess-command-output*")))
         (proc (ess-command--get-proc proc no-prompt-check))
         (sentinel (inferior-ess--output-sentinel))
-        (timeout (or timeout 1)))
+        (timeout (or timeout ess--command-default-timeout)))
```

Comment:
doesn't matter much at al, but wouldn't `ess-command--default-timeout` be more appropriate?

Reply:
I briefly pondered treating `ess-command-` as a subnamespace but it doesn't really work with `ess-string-command` or `ess-boolean-command`. I wish we had everything under `ess-command-` so this related functionality would be more easily discovered.

Reply:
Hmm. I tested locally with emacs devel. Looking into it.